File: frontend/src/services/billService.js
import api from './api';

export const getAllBills = () => api.get('/bills');
export const getBillById = (id) => api.get(`/bills/${id}`);
export const createBill = (bill) => api.post('/bills', bill);
export const updateBill = (id, bill) => api.put(`/bills/${id}`, bill);
export const deleteBill = (id) => api.delete(`/bills/${id}`);

File: backend/src/main/java/com/hospital/service/BillService.java
package com.hospital.service;

import com.hospital.entity.Bill;
import java.util.List;

public interface BillService {
    Bill createBill(Bill bill);
    Bill getBillById(Long id);
    List<Bill> getAllBills();
    Bill updateBill(Long id, Bill billDetails);
    void deleteBill(Long id);
}
